src/mlpack/tests/gmm_test.cpp

```diff
@@ -198,7 +200,9 @@ BOOST_AUTO_TEST_CASE(GMMTrainEMMultipleGaussians)
       // Calculate the actual means and covariances because they will probably
       // be different (this is easier to do before we shuffle the points).
       means[i] = arma::mean(data.cols(point, point + counts[i] - 1), 1);
-      covars[i] = ccov(data.cols(point, point + counts[i] - 1), 1 /* biased */);
+      arma::uword norm_type = 1;
+      arma::mat sub = data.cols(point, point + counts[i] - 1);
```

Comment:
No overload function for `ccov(subview<double> ,int )` and hence had to type case it.

Reply:
looks ok

Reply:
a slightly more compact way might be to do convert to mat directly in the ccov() call (as shown below), but either way works

    covars[i] = mlpack::math::ccov( mat(data.cols(point, point + counts[i] - 1)), 1 /* biased */);